blpapi-bind.ts

```diff
@@ -0,0 +1,25 @@
+/// <reference path="./typings/tsd.d.ts" />
+
+import path = require('path');
+import events = require('events');
+
+// Somewhat hacky but needed to import the 'blpapijs' module.
```

Comment:
After consult with the typescript guys(Microsoft/TypeScript#2712), found a better way to import the js typing information into typescript.

1. Change `blpapijs.d.ts` as posted below.
2. Move `blpapijs.d.ts` to `./build/Release/`(we need to add this cp command into make)
3. Remove reference to `blpapijs.d.ts` in `/// <reference path="./typings/tsd.d.ts" />`

blpapijs.d.ts
```typescript
import events = require('events');

declare module blpapijs {

    interface ISessionOpts {
        serverHost?: string;
        serverPort?: number;
        authenticationOptions?: string;
    }

    interface ISubscription {
        security: string;
        fields: string[];
        options?: any;
        correlation: number;
    }

    interface IIdentity {
    }

    class Session extends events.EventEmitter
    {
        constructor(opts: ISessionOpts);

        start(): Session;

        stop(): Session;

        destroy(): Session;

        openService(uri: string, cid: number): number;

        subscribe(sub: ISubscription[], identity?: IIdentity, label?: string): Session;

        resubscribe(sub: ISubscription[], label?: string): Session;

        unsubscribe(sub: ISubscription[], label?: string): Session;

        request(uri: string, name: string, request: any, cid: number,
                identity?: IIdentity, label?: string): number;

        createIdentity(): IIdentity;

        generateToken(cid: number): number;

        sendAuthorizationRequest(token: string, identity: IIdentity,
                                 cid: number): number;
    }
}

export = blpapijs;
```